swarm_copy/app/dependencies.py

```diff
@@ -36,7 +38,54 @@ async def get_openai_client(
             await client.close()
 
 
-def get_starting_agent(settings: Annotated[Settings, Depends(get_settings)]) -> Agent:
+def get_connection_string(
```

Comment:
Since this function only depends on the `Settings`, wouldn't it make more sense to directly turn it into a method of the `Settings` class?

Reply:
Hm, it seems to me that this function is not logically related to the Settings class (but maybe I am wrong) other than the fact that it is using a Settings instance. I would not move it into that class personally.

Reply:
Fair:) Not a must:) Arguably subjective. Resolving conversation